src/Compilers/VisualBasic/Test/Syntax/TestSyntaxNodes.vb

```diff
@@ -2796,6 +2796,25 @@ End Module
             Assert.Throws(Of ArgumentNullException)(Sub() newTree.GetChangedSpans(BlankTree))
         End Sub
 
+        <Fact>
+        Public Sub TestSyntaxTree_GetChangesWhenSubChangedToFunction()
```

Comment:
>TestSyntaxTree_GetChangesWhenSubChangedToFunction [](start = 19, length = 49)

it looks like VisualBasicSyntaxDiffer supports more scenarios that aren't covered yet. Could you add corresponding tests? Thanks